src/main/kotlin/net/atos/zac/app/audit/converter/RESTHistorieRegelConverter.kt

```diff
@@ -47,9 +52,18 @@ class RESTHistorieRegelConverter @Inject constructor(
 
     private fun convertAuditTrailBasis(historieRegel: RESTHistorieRegel, auditTrailRegel: AuditTrailRegel) =
         historieRegel.apply {
+            actie = convertActie(auditTrailRegel.actie)
+            applicatie = auditTrailRegel.applicatieWeergave
             datumTijd = auditTrailRegel.aanmaakdatum
             door = auditTrailRegel.gebruikersWeergave
-            applicatie = auditTrailRegel.applicatieWeergave
             toelichting = auditTrailRegel.toelichting
         }
+
+    private fun convertActie(auditTrailActie: String): RESTHistorieActie? =
+        when (auditTrailActie) {
+            CREATE -> RESTHistorieActie.GEKOPPELD
+            UPDATE, PARTIAL_UPDATE -> RESTHistorieActie.GEWIJZIGD
+            DESTROY -> RESTHistorieActie.ONTKOPPELD
```

Comment:
Not sure yet what the mono repo pattern is; but this conversion could also be provided in frontend; then I would use the translations for it, since it is needed just as presentation information. But all fine ofc to keep it here if that is main pattern.

Reply:
I would prefer to have it in the backend for several reasons:
* we have enum as return type. We could change that to string, but it feels better to have known actions
* some of the data is not available in frontend (original OpenZaak objects) as we provide just a specific view (just a glimpse actually) to the frontend
* (in future) we could convert not only based on string but old/new and more